.gitignore

```diff
@@ -0,0 +1,3 @@
+.vagrant
+ansible_config.retry
```

Comment:
This matches retry file only for an ansible playbook named "ansible_config". Change this to "*.retry".